gw_eccentricity/load_data.py

```diff
@@ -901,9 +1138,17 @@ def get_params_dict_from_sxs_metadata(metadata_path):
             m2 = float(line.split("=")[-1].strip())
     # numerical noise can make m1 slightly lesser than m2. Catch this
     # whenver it happens
-    if m1 < m2:
-        raise Exception(f"SXS metadata gives m1 = {m1} < m2 = {m2}")
-    params_dict = {"q": m1/m2,
+    q = m1/m2
+    if q < 1:
+        # if q > 0.99, treat it as 1.
+        if q > 0.99:
```

Comment:
As I was saying, 0.99 is not high enough. What's the actual number for q you are getting? Can we make it 1-1e-5?

Reply:
I think it need not be 0.99 and in fact, the difference is very small close to what you are suggesting. So, I will go through the Ecc1d runs since these have q=1 and find the typical value of 1-q

Reply:
And the one that is 1e-4, is q=1-1e-4, and not q=1+1e-4, right?
If yes, let's go with 5e-4 for now, and we can adjust later if needed.

Reply:
Okay, done!